src/Router.jsx

```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+import { sidebar_routes } from './routes-data/sidebar-routes';
```

Comment:
first of all, i'm dissapointed ;p
but...
dissapointed that you start with busness logic in PR about component
you mix business logic (routes of application - business logic) with component
seperate component logic PRs from product backlog logic next time pls ok? (h: understand)